xml/System.Security.Cryptography/CryptoStream.xml

```diff
@@ -58,6 +58,14 @@
 > [!IMPORTANT]
 >  This type implements the <xref:System.IDisposable> interface. When you have finished using the type, you should dispose of it either directly or indirectly by calling its <xref:System.Security.Cryptography.CryptoStream.Clear%2A> method, which in turn calls its <xref:System.IDisposable> implementation. To dispose of the type directly, call its <xref:System.Security.Cryptography.CryptoStream.Clear%2A> method in a `try`/`catch` block. To dispose of it indirectly, use a language construct such as `using` (in C#) or `Using` (in Visual Basic). For more information, see the "Using an Object that Implements IDisposable" section in the <xref:System.IDisposable> interface topic.  
   
+> [!IMPORTANT]
+>  From .NET 6 onwards, a breaking change is implemented on reading <xref:System.Security.Cryptography.CryptoStream>. When `Stream.Read` or `Stream.ReadAsync` is called on one of the affected stream types with a buffer of length `N`, the operation completes when:
+
+> - At least one byte has been read from the stream, or
+> - The underlying stream they wrap returns 0 from a call to its read, indicating no more data is available.
+
+> Also, when `Stream.Read` or `Stream.ReadAsync` is called with a buffer of length 0, the operation succeeds once a call with a non-zero buffer would succeed.
```

Comment:
@gewarren Is our voice here "Also, ", "Additionally, " or something else I can't conceive of pre-coffee?

Reply:
I like Dan's suggestion, so will wait to review grammar until that change is made.